docs/blockchain/why-us-aleph.md

```diff
@@ -0,0 +1,58 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [pyrsia/pyrsia#0000](https://github.com/pyrsia/pyrsia/pull/0000)
+- Pyrsia Issue: [pyrsia/pyrsia#0000](https://github.com/pyrsia/pyrsia/issues/0000)
+
+# Summary
+
+Why we chose to use Aleph for our consensus engine
+
+# Motivation
+
+One of Pyrsia's main design decisions is to use a blockchain for its underlying datastore. The data structure(s) of a
+blockchain are plenty challenging but nothing mind bending. However, the real challenge to blockchain is consensus;
+how do you get a bunch of strangers to trust each other and agree, when some of them might be corrupt and you could
+have any number of faults in the system?
+
+# Our core requirements #
+- No currency. We wish to avoid legal problems
```

Comment:
Cryptocurrency and consensus algorithm are different concepts.

Reply:
True, and I don't believe that I conflated them. Although, One of the mechanisms to encourage concensus participation is "Proof of Work" - and, participants are paid for winning the contest in coinage. So, they are, by design, linked together.

Reply:
that's a little bit of a mischaracterization... having a reward mechanism is not at all related to PoW.

"incentivising" can also be a negative affect like "Proof of Burn"


I think it's better to phrase this

```suggestion
- Avoid any notion of monetary assets (e.g balance, account, tokens)
```